docs/pages/enroll-resources/database-access/enroll-aws-databases/rds-oracle.mdx

```diff
@@ -0,0 +1,378 @@
+---
+title: Database Access with AWS RDS Oracle with Kerberos Authentication
+description: How to configure Teleport Database Access with AWS RDS Oracle with Kerberos authentication.
+---
+
+(!docs/pages/includes/database-access/db-introduction.mdx dbType="Amazon RDS for Oracle" dbConfigure="with Kerberos authentication"!)
+
+## How it works
+
+The Teleport Database Service joins the same Active Directory domain as the Oracle
+database and uses the Kerberos protocol to authenticate with Oracle.
+When a user connects to Oracle via Teleport, the Database Service obtains a
+service ticket from Active Directory, then uses a long-term key for the database
+user to decrypt the ticket and connect to Oracle. At that point, the
+Database Service forwards user traffic to the database.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- An AWS account with the necessary permissions to create RDS instances and AWS Directory Service directories.
+- An Amazon RDS for Oracle database instance.
+- An AWS Directory Service Managed Microsoft AD.
+- A Linux node joined to the same Active Directory domain as the database. This
+  guide will walk you through the joining steps if you don't have one.
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/9. Prepare the Oracle RDS Instance
+
+Before configuring Teleport, ensure your Oracle RDS instance has Kerberos authentication and TLS properly configured:
+
+1. Follow the [AWS RDS Oracle Kerberos Setup guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/oracle-kerberos-setting-up.html) to enable Kerberos authentication on your instance.
+
+2. Enable TLS on your Oracle RDS instance by following the [AWS RDS Oracle SSL Setup documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.Oracle.Options.SSL.html). Ensure `SQLNET.SSL_VERSION` is set to `1.2` for optimal security.
```

Comment:
any cipher we want to recommend? or just leave it as default

Reply:
I think the default is fine. I also don't want to suggest that we mandate any particular choice.